### ISSUE TITLE
H4H Link Tab-Focus Support for Accessibility

### ISSUE THREAD

**Issue by rileywong311** [SCUACM/ACM-Website#39]
<img width="242" alt="image" src="https://github.com/SCUACM/ACM-Website/assets/114180322/6df3740b-389c-494e-b69a-fe6f755dd138">

The H4H link cannot be tab-focused (pressing the `tab` key to select website elements). This is an accessibility feature that should be implemented. 

I believe this can be accomplished with a new CSS `:focus` selector style similar to how `:hover` works.

https://www.w3schools.com/cssref/sel_focus.php
